web/src/layouts/components/bars/breadcrumb/index.tsx

```
@@ -81,42 +81,46 @@ export default defineComponent({
+                                          await router.push({path: item.redirect as string})
+                                        } else if (isComponent && item.name !== route.name) {
+                                          await router.push({path: item.path})
+                                        }
+                                      }}
+                                      v-slots={{
+                                        'default': () =>
+                                          <span>{item?.meta?.i18n ? useTrans(item?.meta?.i18n) : item?.meta?.title}</span>,
+                                        'prefix-icon': () => item?.meta?.icon
+                                          && <ma-svg-icon name={item?.meta?.icon} size={18}/>,
+                                      }}
+                                    />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)